go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/containernetworking/cni v0.8.1
-	github.com/containers/common v0.41.1-0.20210721112610-c95d2f794edf
+	github.com/containers/common v0.41.0
```

Comment:
That is reverting changes.  We need to cut a v0.42.0.

Reply:
(v0.42.0 has been tagged)

Reply:
bumped and repushed, thanks!